codice/ddf
# DDF-2316: Windows and OWASP fix (back-port)

#### What does this PR do?
Back-ports the Windows build and OWASP fix. 

#### Choose 2 committers to review/merge the PR.
@figliold 
@shaundmorris

#### How should this be tested?
Already done. Windows builds should pass when done correctly. See the following PR's: 
https://github.com/codice/ddf/pull/1179
https://github.com/codice/ddf/pull/1126
https://github.com/codice/ddf/pull/1115
